automated-comments/go/hamming/missing_distance_function.md

```diff
@@ -0,0 +1 @@
+The `Distance` function is missing or couldn't be found.
```

Comment:
Maybe just 'is missing'? Missing and can't be found are really more or less the same thing, aren't they? Possibly the student forgot to upload the solution file, and just submitted the tests instead (this happens surprisingly often). We might give them a hint if that's the case.